test-app/tests/acceptance/profiles-view-test.js

```diff
@@ -1,15 +1,12 @@
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
-import FactoryGuy, {
-  makeList,
-  mockFindAll,
-  setupFactoryGuy,
-} from 'ember-data-factory-guy';
+import FactoryGuy, { makeList, mockFindAll } from 'ember-data-factory-guy';
 import { visit } from '@ember/test-helpers';
+import { inlineSetup } from '../helpers/utility-methods';
 
 module('Acceptance | Profiles View', function (hooks) {
   setupApplicationTest(hooks);
-  setupFactoryGuy(hooks);
+  inlineSetup(hooks, '-json-api');
```

Comment:
We deleted the default application adapter/serializer for the test-app (in a previous PR). These were JSONAPI.

In 5.3 it no longer provides a default application adapter/serializer for models, so gotta define one, so we use the inline setup to do so